src/js/app.js

```diff
@@ -14,36 +14,41 @@ const app = new Vue({
     input: '',
 
     include_similar: false,
-    sort_options: SORT_OPTIONS,
+    sort_options: SORT_OPTIONS_SHORT,
     sort: 'stroke',
 
     candidates: [],
     radical_selection: undefined,
 
     searching: false,
+
+    isDropdownOpen: false,
+    isIncludeSimilar: false,
   },
   template: `
     <div class="app">
-      <div>
-        <input type="text" v-model="input" v-on:change="lookup">
+      <div class="search">
+          <div class="search_sort" v-on:click="openDropdown" v-bind:class="{ open: isDropdownOpen}">
```

Comment:
Ah, sorry, all of my HTML was indented 2-spaces, but I think yours is 4-space.

Let's consolidate!